package.json

```diff
@@ -30,64 +30,56 @@
   },
   "devDependencies": {
     "babel-preset-env": "^1.2.2",
+    "babel-preset-es2015": "^6.24.1",
+    "babel-register": "^6.24.1",
     "babelify": "^7.3.0",
+    "browser-sync": "^2.18.8",
+    "browserify": "^14.3.0",
+    "browserify-incremental": "^3.1.1",
+    "del": "^2.2.2",
     "eslint": "3.18.0",
```

Comment:
Jep. I’m open for discussion.
I don’t see the need for pinning the dependencies (Using `x.x.x` instead of `^x.x.x`) since we now have reproducible installs with Yarn lockfiles and [npm5](https://github.com/npm/npm/pull/16244) will make [shrinkwrap great again ](https://twitter.com/ReBeccaOrg/status/854094958260699136)